report/2.Analysis/ArduinoInEducation.tex

```diff
@@ -0,0 +1,50 @@
+
+This technical oriented course, often called electronic\footnote{The name varies between institutions}, aims to familiarize the students with electrical components and circuits as well as basic programming.\cite{holstebro_electronic}
+Arduino is an ideal platform for this, as it is possible to connect I/O devices to the Arduino micro-controller, which in turn can be programmed to do simple tasks. 
+This course is then used to both spike the interest of the student regarding engineering fields, such as programming or electronics.
+
+\subsection{HTX CAO Electronic Course}
+In relation to the discipline of understanding the initial problem. The project group contacted one of the major HTX institutions in North Jutland, Denmark. Namely HTX CAO. The following subsection will summarize the important and noticable aspects of the electronic course conducted on HTX CAO. The conclusions reached in this subsection are purely based on the statements made by the students and the teacher of this specific electronics course.
+
+The electronics course itself does not teach the students programming directly. Instead it focuses on project based arduino development. All educational programming experience of the students (if any), originates from another optional course called "Programming". The Programming course teaches the students the basic contructions and termonology of general programming. it utilizes the languages C{\#}. 
+
+The electronics course focuses on engaging the students in making arudino projects, not in making the students understand the theory behind the projects. This means that the students both have a very limited understanding of the arudino platform's architecture and a low understanding of the arduino language explained earlier in the report, which they utilize in the electronics course on HTX CAO.
+
+Because of all this the electronics course have a relatively high learning curve. Which results in a high number of students abandoning the course in favor of some of the courses with a more specificly defined theory structures.
+
+Clearly there is a majojr gap between the students who are interested in programming or electronics and have no prior experience and students who have prior experiences. 
+Based on students' and the teacher's statements it seems like the course needs a better gateway into the arduino platform, the programming aspects of the course or both.
+
+A substantial number of the students in the class expressed their frustation with relative complexity of the arduino language. The teacher supplemented with saying that the language is not necessary complex, but the time spent on the different concepts are far too short, which is a result of the education being a junior education and not a college/university level education.
```

Comment:
Change "with relative complexity" to "with the relative complexity". Change "necessary" to "necessarily". Change "concepts are far too short" to "concepts is far too short"